src/prefect/backend/flow_run.py

```diff
@@ -58,7 +66,14 @@ def watch_flow_run(
     flow_run = FlowRunView.from_flow_run_id(flow_run_id)
 
     if flow_run.state.is_finished():
-        output_fn(logging.INFO, "Your flow run is already finished!")
+        time_ago = (pendulum.now() - flow_run.updated_at).as_interval().in_words()
```

Comment:
wut

Reply:
where has this been all my life

Reply:
@zangell44 fyi this can actually be written as `flow_run.updated_at.diff_for_humans()` which is way cooler